image_explorer/public/js/image_explorer_edit.js

```diff
@@ -1,10 +1,11 @@
-function ImageExplorerEditBlock(runtime, element) {
+ function ImageExplorerEditBlock(runtime, element) {
     var xmlEditorTextarea = $('.block-xml-editor', element),
         xmlEditor = CodeMirror.fromTextArea(xmlEditorTextarea[0], { mode: 'xml', lineWrapping: true });
 
     $(element).find('.save-button').bind('click', function() {
         var data = {
             'display_name': $(element).find('.edit-display-name').val(),
+            'hot_spots': changeBooleanVal($('.edit-hot-spots', element).is(':checked')),
```

Comment:
We should not be changing this to a string - keep it a javascript boolean, so that it can get properly (de)serialized by JSON.

Reply:
This should be fine:

```javascript
'hot_spots': $('.edit-hot-spots', element).is(':checked')
```